tests/api/test_super_study.py

```diff
@@ -38,11 +38,12 @@ def test_details(self, client, super_study):
         assert rsp['data']['type'] == 'super-studies'
         assert rsp['data']['id'] == '1'
         _attr = rsp['data']['attributes']
-        assert len(_attr) == 5
+        assert len(_attr) == 6
         assert _attr['super-study-id'] == 1
         assert _attr['title'] == 'Human Microbiome'
         assert _attr['description'] == 'Just a test description'
         assert _attr['biomes-count'] == 1
+        assert _attr['url-slug'] == 'human-micro'
 
```

Comment:
Maybe add a second test exactly the same as this one, but instead of using `args=['1']` in line:31 uses `args=['human-micro']`. Or modify the current one to do both requests